tests/integration/export_v2/conftest.py

```diff
@@ -297,7 +297,7 @@ def configured_project_with_ontology(client, initial_dataset, ontology,
 
 
 @pytest.fixture
-def configured_project_without_data_rows(client, ontology, rand_gen):
+def configured_project_with_one_data_row(client, ontology, rand_gen):
```

Comment:
this does not create a data row, should we revert the rename?

Reply:
another good find